atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -19,12 +19,14 @@
 import java.util.List;
 import java.util.Set;
 
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.google.common.collect.Sets;
 
 public class AssertUtils {
 
```

Comment:
Minor: I'd prefer the loger on line 28 and the blank line on line 29 to separate it from the method declaration.